### PR TITLE
Update appleseed.package.py, cleanups

### PR DESCRIPTION
**Pull request by dictoon**

### CODE REVIEW

File: scripts/travis/build-macos.sh [hunk 1]
```diff
@@ -174,8 +174,8 @@ cmake \
     -DOSL_QUERY_LIBRARY=$THISDIR/lib/liboslquery.dylib \
     -DOSL_COMPILER=$THISDIR/bin/oslc \
     -DOSL_QUERY_INFO=$THISDIR/bin/oslinfo \
-    -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/include/python2.7/ \
-    -DPYTHON_LIBRARY=/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib \
+    -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/include/python2.7/ \
```
Comment (pjessesco) [inline scripts/travis/build-macos.sh]: We can use `/usr/local/opt/python@2/Frameworks/Python.framework/Versions/2.7/include/python2.7`, similar with line 614 in `appeseed.package.py`. 

In my mac, python is installed in `/usr/local/Cellar/python@2/2.7.16`, not `2.7.16_1`.

File: scripts/travis/build-macos.sh [hunk 2]
```diff
@@ -174,8 +174,8 @@ cmake \
...
+    -DPYTHON_LIBRARY=/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib \
```
Comment (pjessesco) [inline scripts/travis/build-macos.sh]: Same here.

File: scripts/travis/build-macos.sh (hunk 1 above)
Comment (pjessesco) [reply scripts/travis/build-macos.sh]: Is it changed because this script is for Travis?

Comment (dictoon) [reply scripts/travis/build-macos.sh]: That's a very good point!

Indeed, `/usr/local/opt/python@2` links to `/usr/local/Cellar/python@2/2.7.16_1`:

```
Franzs-Mac:scripts franz$ ll /usr/local/opt/python@2
lrwxr-xr-x  1 franz  admin  27 Sep  5 19:30 /usr/local/opt/python@2 -> ../Cellar/python@2/2.7.16_1
```

Comment (dictoon) [reply scripts/travis/build-macos.sh]: Made the modification, let's see if Travis/macOS passes.

**Pull request merged**